### PR TITLE
Cyclus

### PR DESCRIPTION
**Pull request by bam241**
I have make some correction of ex  (simulation lengh.. and upate the fuel output recipe)
I add also CYCLUS input for ex 2 & 3

I also add some results in cvs format (probably not standard yet, and should be updated... but this is still something...)

### CODE REVIEW

File: input/OFCB/1/UnBatched/1.xml [hunk 1]
```diff
@@ -102,15 +102,35 @@
...
 
-</simulation>
\ No newline at end of file
+</simulation>
+
+
+
+
+
+
```
Comment (scopatz) [inline input/OFCB/1/UnBatched/1.xml]: What is with the whitespace?  Is it on sale? :)

**Comment by bam241** [comment]
@scopatz yes, it was: buy one get 1 free!

Because you don't like them, I have just resale them... for twice the rices :)

File: input/OFCB/1/Batched/Output.txt [hunk 2]
```diff
@@ -0,0 +1,1223 @@
+Time[month],reactor_In[kg],Reactor_Out[kg],
```
Comment (scopatz) [inline input/OFCB/1/Batched/Output.txt]: This file seems to be in CSV format, so it should have a `*.csv` extension.

File: input/OFCB/1/UnBatched/1.xml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    </facility>
+
+    <facility>
+        <name>Reactor</name>
+        <config>
+            <Reactor>
+                <fuel_inrecipes>    <val>UOX</val>        </fuel_inrecipes>
+                <fuel_outrecipes>    <val>SpentUOX</val>    </fuel_outrecipes>
+                <fuel_incommods>    <val>UOX</val>        </fuel_incommods>
+                <fuel_outcommods>    <val>SpentUOX</val>    </fuel_outcommods>
```
Comment (scopatz) [inline input/OFCB/1/UnBatched/1.xml]: Weird internal spacing on these lines.

File: input/OFCB/1/UnBatched/Output.txt [hunk 4]
```diff
@@ -0,0 +1,1223 @@
+Time[month],reactor_In[kg],Reactor_Out[kg],
```
Comment (scopatz) [inline input/OFCB/1/UnBatched/Output.txt]: `*.csv`

File: input/OFCB/2/Batched/2.xml [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+    </facility>
+
+    <facility>
+        <name>Reactor</name>
+        <config>
+            <Reactor>
+                <fuel_inrecipes>    <val>UOX</val>        </fuel_inrecipes>
+                <fuel_outrecipes>    <val>SpentUOX</val>    </fuel_outrecipes>
+                <fuel_incommods>    <val>UOX</val>        </fuel_incommods>
+                <fuel_outcommods>    <val>SpentUOX</val>    </fuel_outcommods>
```
Comment (scopatz) [inline input/OFCB/2/Batched/2.xml]: spacing issue.

File: input/OFCB/4/UnBatched/4.xml [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+                    <val>1</val>
+                    <val>1</val>
+                    <val>1</val>
+                    <val>1</val>
+                </n_build>
+            </DeployInst>
+        </config>
+    </prototype>
+
+    <agent> <name>myDeployer</name> <prototype>deployer</prototype> </agent>
```
Comment (scopatz) [inline input/OFCB/4/UnBatched/4.xml]: camel case? Also `myDeployer` isn't an illustrative name.

Comment (bam241) [inline input/OFCB/4/UnBatched/4.xml]: @scopatz I don't get the `camel case?` :s

I have corrected the rest :)

**Comment by scopatz** [comment]
I don't trust camels since they are known deserters :).

**Comment by scopatz** [comment]
Looks good, thanks.

**Pull request merged**